enterprise/templates/enterprise/_enterprise_customer_sidebar.html

```diff
@@ -0,0 +1,7 @@
+<img class="enterprise-logo" src="{{ enterprise_customer.branding_configuration.logo.url }}" alt="{{ enterprise_customer.name }}"/>
+<p class="partnered-text">
+    {{ welcome_text }}
+</p>
+<p class="partnered-text">
+    {% autoescape off %}{{ enterprise_welcome_text }}{% endautoescape %}
+</p>
```

Comment:
Can you also include this partial in [grant_data_sharing_permissions.html](https://github.com/edx/edx-enterprise/blob/master/enterprise/templates/enterprise/grant_data_sharing_permissions.html#L27)?

Reply:
Done.